test/plots/barPlotTests.ts

```diff
@@ -724,20 +724,20 @@ describe("Plots", () => {
 
     describe("Horizontal Bar Plot label visibility", () => {
       let svg: d3.Selection<void>;
-      let yScale: Plottable.Scales.Category;
+      let yScale: Plottable.Scales.Linear;
       let xScale: Plottable.Scales.Linear;
-      let barPlot: Plottable.Plots.Bar<number, string>;
+      let barPlot: Plottable.Plots.Bar<number, number>;
       beforeEach(() => {
         svg = TestMethods.generateSVG(600, 400);
-        yScale = new Plottable.Scales.Category().domain(["A", "B"]);
+        yScale = new Plottable.Scales.Linear();
         xScale = new Plottable.Scales.Linear();
 
         let data = [
-          {y: "A", x: -1.5},
-          {y: "B", x: 1},
+          {y: 1, x: -1.5},
+          {y: 2, x: 1},
         ];
 
-        barPlot = new Plottable.Plots.Bar<number, string>(Plottable.Plots.Bar.ORIENTATION_HORIZONTAL);
```

Comment:
We change the test because we need to set the domain so that the label is partially cut off.